snuba/datasets/storages/groupassignees.py

```diff
@@ -36,14 +36,14 @@
     columns=columns,
     local_table_name="groupassignee_local",
     dist_table_name="groupassignee_dist",
-    storage_set_key=StorageSetKey.EVENTS,
+    storage_set_key=StorageSetKey.CDC,
```

Comment:
How do we deal with backwards compatibility for on premise customers for those customers who could have customized their storage set deployment configuration assigning different storage sets to different clickhouse clusters ?

Reply:
The CDC storage set is new and the storages it covers were previously part of EVENTS storage set so it's impossible that they could have been previously assigned to different ClickHouse clusters by anyone. The EVENTS_RO storage set on the other hand could theoretically have been assigned different dist clusters? That would be kind of weird though and I didn't handle this case since it seemed like a fairly unlikely edge case. I wanted to start enforcing the this constraint now so that no one can do stuff like this with EVENTS_RO in the future.

Reply:
Right but they would have a custom cluster configuration. The moment they upgrade snuba and get the new storageSet, that will not be in their config and Snuba won't start.

Reply:
Yes, they just need to add it to their settings. This is a standard process for anytime we add a storage set, which we are supposed to be able to do easily (though we didn't in a while). Having said that running ClickHouse in onpremise in any way other than a single node is currently not really supported or documented so I also don't think we need to dwell on this. What are the concerns specifically?